### ISSUE TITLE
TextBox FontFamily FallBack

### ISSUE THREAD

**Issue by GihanSoft** [AvaloniaUI/Avalonia#4708]
Setting Font Family FallBack in WPF is like this ([link](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.fontfamily?view=netcore-3.1#defining-a-font-fallback-sequence-in-code))
```xaml
<TextBlock FontFamily="Comic Sans MS, Verdana">Hello, world</TextBlock>
```
this not work in AvaloniaUI, instead this works:
```xaml
<TextBlock FontFamily="'Comic Sans MS', 'Verdana'">Hello, world</TextBlock>
```
I tried in multi language fonts (Lucida Console & B Yekan)

**Comment by GihanSoft** [comment]
> this not work in AvaloniaUI, instead this works:
> 
> ```
> <TextBlock FontFamily="'Comic Sans MS', 'Verdana'">Hello, world</TextBlock>
> ```
This is not working either

**Comment by Gillibald** [comment]
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Skia/Avalonia.Skia/FontManagerImpl.cs#L75

**Comment by Gillibald** [comment]
How are you testing the fallback?

**Comment by GihanSoft** [comment]
@Gillibald 
![Capture2](https://user-images.githubusercontent.com/35791547/93768667-9d864280-fc26-11ea-9393-0d88a975be74.PNG)
![Capture3](https://user-images.githubusercontent.com/35791547/93768674-9f500600-fc26-11ea-9af0-0bc0e1d69117.PNG)
![Capture](https://user-images.githubusercontent.com/35791547/93768675-a0813300-fc26-11ea-93ca-e44459ed9069.PNG)

**Comment by Gillibald** [comment]
TextBox does not support any font fallback. This will change later this year.

**Comment by GihanSoft** [comment]
I think if you set fallback for each char it will word

**Comment by Shayan-To** [comment]
> 
> 
> TextBox does not support any font fallback. This will change later this year.

Is this done yet?

**Comment by Gillibald** [comment]
#4893

**Comment by GihanSoft** [comment]
> #4893

I found out there is a thing named bidi or bidirectional engine you should proceed text before sending it to harfbuzz. For complex texts. You probably know this already.

**Comment by Shayan-To** [comment]
@Gillibald Any estimate of when this will be done?

**Comment by Gillibald** [comment]
Somewhere in April. I don't have any free time at the moment. The current state is testable but still needs some major reworks.

**Comment by Takoooooo** [comment]
Fixed in #4893.
